indexer/ui/schema.prisma

```diff
@@ -4,106 +4,203 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
+  url      = "postgresql://db_username:db_password@localhost:5434/db_name"
```

Comment:
```suggestion
  url      = env("DATABASE_URL")
```
should keep using DATABASE_URL env var. Might want to update script to be smart enough to do this